test/e2e/scheduling/priorities.go

```diff
@@ -457,20 +463,11 @@ func addRandomTaitToNode(cs clientset.Interface, nodeName string) *v1.Taint {
 func updateNodeAllocatable(c clientset.Interface, nodeName string, memory, cpu int64) error {
 	node, err := c.CoreV1().Nodes().Get(nodeName, metav1.GetOptions{})
 	framework.ExpectNoError(err)
-	oldData, err := json.Marshal(node)
 	if err != nil {
 		return err
 	}
 	node.Status.Allocatable[v1.ResourceMemory] = *resource.NewQuantity(memory, resource.BinarySI)
 	node.Status.Allocatable[v1.ResourceCPU] = *resource.NewMilliQuantity(cpu, resource.DecimalSI)
-	newData, err := json.Marshal(node)
-	if err != nil {
-		return err
-	}
-	patchBytes, err := strategicpatch.CreateTwoWayMergePatch(oldData, newData, v1.Node{})
-	if err != nil {
-		return err
-	}
-	_, err = c.CoreV1().Nodes().Patch(string(node.Name), types.StrategicMergePatchType, patchBytes)
+	_, err = c.CoreV1().Nodes().UpdateStatus(node)
```

Comment:
Is that to say, `Patch()` doesn't work for API fields in status?

Reply:
> Is that to say, Patch() doesn't work for API fields in status? 
Yeah, that's right. Patch wasn't working for updating cpu or memory. 

In order to test it, I added taints to node and applied patch, this worked while cpu and memory were not getting updated. So, I had to use UpdateStatus.